src/status_im/ui/screens/browser/views.cljs

```diff
@@ -132,7 +133,9 @@
         ;; https://reactjs.org/docs/events.html#event-pooling
         :on-message                                 #(re-frame/dispatch [:browser/bridge-message-received (.. ^js % -nativeEvent -data)])
         :on-load                                    #(re-frame/dispatch [:browser/loading-started])
-        :on-error                                   #(re-frame/dispatch [:browser/error-occured])
+        :on-error                                   #(let [native-event (.. ^js % -nativeEvent)
+                                                           _ (log/info "### on-error" native-event)]
```

Comment:
Do we want to keep this log entry? Can it be moved to `:browser/error-occured`?

Reply:
Sorry, this was for debugging only, forgot to remove. Done